src/Events/Integrations/Plugins/Event_Tickets/Emails/Template.php

```diff
@@ -20,7 +20,7 @@ class Template extends Tribe__Template {
 	 */
 	public function __construct() {
 		$this->set_template_origin( TEC::instance() );
-		$this->set_template_folder( 'src/views/v2/emails' );
+		$this->set_template_folder( 'src/views/integrations/event-tickets/emails' );
```

Comment:
This seems odd, as we respect that in ET for TEC templates. It'll be the first time we have something outside.

Reply:
We need to start moving things out of V2. That was there just for the Views V2 project.

Reply:
I understand. I mention this because if I were a user making changes to the templates, I'll find the ET ones in the `v2` folder (where we have the tickets commerce templates, the tickets, and RSVP blocks). Then when I'm trying to find the TEC templates for emails they'll be in a folder that doesn't follow the pattern.

Reply:
I know it's a bit weird, I would have loved to change the ones from ET too, but I feel like it would be too last minute. 

It's super important that we start moving away from V2 folder.